src/ml/mlApi.ts

```diff
@@ -29,3 +29,33 @@ export async function getModel(projectId: string, modelId: string): Promise<Fire
   });
   return res.response.body;
 }
+
+/**
+ * List Firebase models.
+ * @param projectId The project to list the models for
+ * @param options The listing options including filter.
+ * @return The models
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export async function listModels(projectId: string, options: any): Promise<FirebaseModel[]> {
+  const models: FirebaseModel[] = [];
+  const getNextPage = async (pageToken?: string): Promise<void> => {
+    const pageResponse = await api.request("GET", `/${VERSION}/projects/${projectId}/models`, {
```

Comment:
What happens to errors here?  Are they just ignored and signaled by returning a 0-length array?  Would prefer if the call failed.

Reply:
I have a test that shows when the api.request fails you get a call failure. 
It's called "should throw FirebaseError if any call returns an error" under "listModels" in mlApi.spec.ts